scripts/reindex_articles.coffee

```diff
@@ -0,0 +1,22 @@
+require('node-env-file')(require('path').resolve __dirname, '../.env')
+mongojs = require 'mongojs'
+path = require 'path'
+{ indexForSearch } = Save = require '../api/apps/articles/model/save'
+Article = require '../api/apps/articles/model/index'
+search = require '../api/lib/elasticsearch'
+
+# Setup environment variables
+env = require 'node-env-file'
+switch process.env.NODE_ENV
+  when 'test' then env path.resolve __dirname, '../.env.test'
+  when 'production', 'staging' then ''
+  else env path.resolve __dirname, '../.env'
+
+# Connect to database
+db = mongojs(process.env.MONGOHQ_URL, ['articles'])
+
+db.articles.find({ }).toArray (err, articles) ->
```

Comment:
Looks like the main use case is to sync to ES upon save/delete, so probably not a blocker—but doing this will load all of the articles in memory and iterate through them. To work around this it's often [streams](https://github.com/mafintosh/mongojs#streaming-cursors) are helpful, but more likely this would involve some kind of `limit` and iterating in batches.